spk/lirc/Makefile

```diff
@@ -12,7 +12,7 @@ $(error Sorry, this package does not support the $(ARCH) architecture.)
 endif
 
 # Use older LIRC build for ARCHs with kernel < 2.6.36
-ifneq ($(findstring $(ARCH),alpine armada370 armada375 armada38x armadaxp avoton braswell bromolow cedarview evansport x64 x86),$(ARCH))
+ifeq ($(findstring $(ARCH),88f6281 $(PPC_ARCHES) armada370 armadaxp bromolow cedarview evansport qorig),$(ARCH))
```

Comment:
Kernel version does not only depend on architecture but also target DSM version... `qoriq` (not `qorig`) is included in `PPC_ARCHES`.
If you agree, I will get your commits and rework some details to get it compile on all architectures and at least on DSM 5.2 and 6.x

Reply:
sorry for the qoriq misstype,...
yes please add the rework for TC version dependencies.
The only arch with kernels that make a difference for lirc in this PR is in armada370 (2.6.32 until DSM 5.2 and now 3.x, I didn't analyze DSM 6.0.2)
Other candidates for different lirc versions are only 88f6281, bromlow, cedarview and evansport. Maybe you make the fix here for armada370 and will adapt it for the archs not in this PR.

Reply:
there are no TC version dependecies...
but I made a mistake here:
`ifneq ($(findstring $(ARCH),88f6281 $(PPC_ARCHES)),)`
is not the same as
`ifeq ($(findstring $(ARCH),88f6281 $(PPC_ARCHES)),$(ARCH))`
fixed with commit 528ffe60ef9e568225dafcea0cf631d08edc6360